tests/flask/persistance/app.py

```diff
@@ -0,0 +1,19 @@
+from flask import Flask
+from strawberry.flask.persistance.view import ApolloGraphQLPersistanceView
+
+from ..schema import Query, schema
+
+
+def create_app(**kwargs):
```

Comment:
Can we share the `persistence.app` in all the flask tests? or there is something particular we have to set for persistence's tests?

Reply:
`ApolloGraphQLPersistanceView` is specific for using persistance on the Flask view. Since the `query` is not always in the request (when the hash is only provided for example), `parse_request_data` will throw an error. This is why I couldn't use extensions.